brainiak/fcma/mvpa_voxelselector.py

```diff
@@ -38,96 +37,37 @@ class MVPAVoxelSelector:
     Parameters
     ----------
 
-    raw\_data: list of 4D array
-        each element of the list is the raw data of a subject,
-        in the shape of [x, y, z, t]
+    data: 4D array in shape [brain 3D + epoch]
+        contains the averaged and normalized brain data epoch by epoch.
+        It is generated by io.prepare_searchlight_mvpa_data
```

Comment:
This would be clearer if you specified the path as in an import statement, i.e., `.io.prepare_searchlight_mvpa_data`.